packages/x-data-grid-pro/src/tests/filtering.DataGridPro.test.tsx

```diff
@@ -1295,6 +1295,69 @@ describe('<DataGridPro /> - Filter', () => {
       await user.keyboard('{ArrowDown}');
       expect(filterCell).toHaveFocus();
     });
+
+    // https://github.com/mui/mui-x/issues/18949
+    it('should not persist header filter operators across different DataGrid instances', () => {
```

Comment:
Sometimes I wonder if we're not adding too many tests and slowing down the test suite for nothing. This is a bug that is unlikely to be re-introduced, imho the test could be avoided.

Reply:
I agree 👍 
WIll remove the test!